### PR TITLE
feat(GH-318): change soon deprecated set-env action

### LINKED ISSUE #318: Update CI action
The `set-env` command is deprecated and will be disabled soon. Please upgrade to using Environment Files.
For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands.

### PR DESCRIPTION
**Pull request by marcodenisi**
Fixes #318.
Removed the `set-env` action ([soon deprecated](https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/)). Used instead [environment files](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files).

#### ✔ Checklist:

- [X] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [ ] Tests for the changes have been added (for bug fixes / features);
- [ ] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

**Comment by mgrachev** [comment]
@marcodenisi 👋 Thank you for your contribution! 🚀

> - [ ] This PR has been added to CHANGELOG.md (at the top of the list);

Please do that 🙂

**Comment by marcodenisi** [comment]
@mgrachev done, should be ok now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### 🚀 Added
 
 ### 🔧 Changed
+- Update CI action [#318](https://github.com/dotenv-linter/dotenv-linter/pull/320)
```
Comment (mgrachev) [inline CHANGELOG.md:11]:
```suggestion
- Change soon deprecated `set-env` action [#320](https://github.com/dotenv-linter/dotenv-linter/pull/320) ([@marcodenisi](https://github.com/marcodenisi))
```

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@marcodenisi Thank you for your help! 👍

🙏 If it’s not difficult for you, please support the project - click on the star on GitHub ⭐️

**Pull request merged**